evergreen.yml

```diff
@@ -385,7 +385,7 @@ functions:
         ./scripts/lamp \
           -d=${distro|not-linux} \
           -DENABLE_JASPER:BOOL=ON \
-          -DGENNY_INSTALL_DIR:PATH=${workdir}/dist \
+          -DGENNY_INSTALL_DIR:PATH=$PWD/dist \
```

Comment:
quoting - `-DGENNY_INSTALL_DIR="$PWD"/dist` (this was latent before but may as well fix now)

Reply:
Question: What does the quoting accomplish here?

Reply:
You don't want to break tokenization if `$PWD` has any spaces.
E.g. if your cwd is literally `;rm -rf /;` (which is allowed on macos since it allows any unicode code-points in filenames) you could end up...messing a couple things up 😸